src/Traits/DifferenceTrait.php

```diff
@@ -308,25 +308,20 @@ public static function fromNow($datetime)
      */
     public function diffForHumans(?ChronosInterface $other = null, bool $absolute = false): string
     {
-        return static::diffFormatter()->diffForHumans($this, $other, $absolute);
+        return static::getDiffFormatter()->diffForHumans($this, $other, $absolute);
     }
 
     /**
-     * Get the difference formatter instance or overwrite the current one.
+     * Get the difference formatter instance.
      *
-     * @param \Cake\Chronos\DifferenceFormatterInterface|null $formatter The formatter instance when setting.
      * @return \Cake\Chronos\DifferenceFormatterInterface The formatter instance.
      */
-    public static function diffFormatter(?DifferenceFormatterInterface $formatter = null): DifferenceFormatterInterface
+    public static function getDiffFormatter(): DifferenceFormatterInterface
```

Comment:
I dont think you can just replace a static API method like this.
You need to keep the old one for BC reasons.

Reply:
Regarding PHPUnit report, its left to add setDiffFormatter() method and update tests.
To preserve BC edit from maintainers will be needed :)